.github/workflows/docker-reusable.yml

```diff
@@ -197,3 +216,52 @@ jobs:
       - name: Docker - Inspect image
         run: |
           docker buildx imagetools inspect ${{ env.DOCKER_REPO }}:${{ steps.meta.outputs.version }}
+
+
+  # Compute Continuous Tests inputs
+  compute-tests-inputs:
+    name: Compute Continuous Tests list
+    # if: ${{ inputs.continuous_tests_list != '' }}
```

Comment:
Also leftover?

Reply:
It was left intentionally, but we can reuse next line as an example.

Deleted, thanks!